setup.cfg

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length=99
+exclude=migrations
```

Comment:
Commits are kinda atomic so I think that its not that bad for now. I would make sure i split it in future PRs.